### PR TITLE
Final User Guide (with all the fixes!)

### PR DESCRIPTION
**Pull request by devinchristianson**

### CODE REVIEW

File: docs/source/user_guide/instructions.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+.. figure:: static/map_view.png
+   :alt: "Example Map view for Registered User"
+   :align: center
+
+   : The map view as it may appear to a Registered User
+
+Importing Files
+==============================
+
+This function allows you to import .tif and .zip files.
```
Comment (NickKania) [inline docs/source/user_guide/instructions.rst:108]: (nitpick) In the overview we say `.tiff`, They are the same thing so I am fine leaving it as.

File: docs/source/user_guide/public_user_instructions.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+If you do not have an account, this section contains all of the information regarding account creation and using the app.
+If you do have an account, see the "Instructions" section of the manual.
+
+Home View
+==============================
+
+This is the home page of the Birdspotter app. Public users are allowed to see the map view of the public datasets.
+
+.. _public_home_view:
+.. figure:: static/home_view.png
```
Comment (NickKania) [inline docs/source/user_guide/public_user_instructions.rst:14]: Can we use an image of the home screen that an actual public user would see. Currently it is showing the view as an admin user.

**Review by NickKania — CHANGES_REQUESTED** [review]

File: docs/source/user_guide/instructions.rst (hunk 1 above)
Comment (devinchristianson) [reply docs/source/user_guide/instructions.rst:108]: Turns out, .tif is correct, so I've updated the overview in an upcoming commit.

**Comment by devinchristianson** [comment]
Closing this PR because it doesn't seem to be hooking into the CI properly (the CI was disconnected & reconnected since this PR was opened initially)

**Pull request closed without merging**